assets/js/main.js

```diff
@@ -72,11 +70,6 @@ function smoothScrollToTheBottomOfThePageWithjQuery() {
 function smoothScrollToTheTopOfThePageWithjQuery() {
```

Comment:
Could you please explain why we need to use a JQuery bit here? Why don't we just have a vanilla JS instead?

Reply:
We could achieve similar result with native JS, but it would require more testing and code. With the time scope we have to finish the project, I decided to go for jQuery solution, as it has been already tested and has cross browser support. We load jQuery anyway to support Bootstrap component library. Do you think we should refactor it to the native js?

Reply:
Great, that makes sense. I also suggest that maybe we should divide the file into several ones for clarity?